website/src/_posts/2019-07-31-beam-2.14.0.md

```diff
@@ -79,6 +79,7 @@ For more information on changes in 2.14.0, check out the
 ### Known Issues
 
 * Do **NOT** use Python MongoDB source in this release. Python MongoDB source [added](https://issues.apache.org/jira/browse/BEAM-5148) in this release has a known issue that can result in data loss. See ([BEAM-7866](https://issues.apache.org/jira/browse/BEAM-7866)) for details.
+* Can't install the Python SDK on macOS 10.15 ([BEAM-8368](https://issues.apache.org/jira/browse/BEAM-8368))
```

Comment:
Technically you can install the SDK without issue, the error arises when you try to import it in your own script. I'm not sure that's a distinction worth making though since its the same from the user perspective.

Reply:
If it's the same from the users perspective then I think we should leave it as is. Then they can visit the issue tracker for details.

But I can also change it if others also prefer the added detail here :)

Reply:
I'm fine with leaving it as is

Reply:
I think this is fine.

@soyrice you are missing a dot at the end of the sentence. Could you add that?

Reply:
@aaltay Done. I also reformatted each one to match the other bullet points.